lib/recurly/invoice.rb

```diff
@@ -201,6 +202,22 @@ def xml_keys
       super - ['currency']
     end
 
+    # Attempts to update the invoice, returning the success of the request.
+    # Raises an error if attempting to create an invoice using this method.
+    #
+    # @return [true, false]
+    # @raise [RuntimeError] Raises error if you attempt to create an invoice.
+    # @example
+    #   invoice = Recurly::Invoice.find('1000')
+    #   invoice.po_number = '1234'
+    #   invoice.save # => true
+    def save
+      if !persisted?
```

Comment:
This is a good use case for `unless`. `unless` is like `if` but it negates the conditional:

```ruby
unless persisted?
  # do something
end
```

Reply:
Good catch